src/clj/authorizer/logic/functions.clj

```diff
@@ -0,0 +1,86 @@
+(ns authorizer.logic.functions
+  (:require
+    [clojure.spec.alpha :as s]
+    [clj-time.core :as t]
+    [clj-time.coerce :as c]))
+
+"""
+1. The transaction amount should not be above limit\n
+2. No transaction should be approved when the card is blocked\n
+3. The first transaction shouldn't be above 90% of the limit\n
+4. There should not be more than 10 transactions on the same merchant\n
+5. Merchant blacklist\n
+6. There should not be more than 3 transactions on a 2 minutes interval
+"""
+
+(defn approve-card-limit?
+  [{:keys [limit]}
+   {:keys [amount]}]
+  (> limit amount))
+
+(defn approve-card?
+  [{:keys [cardIsActive]}]
```

Comment:
Em clojure agente nao costuma utilizar nomenclatura no estilo Camel case. Entao `cardIsActive` deveria ser `card-is-active` ou melhor ainda apenas `active?`

Reply:
E aqui eu tambem evitaria o destructuring e utilizaria apenas

```clojure
(defn approve-card? [account]
  (:active? account))
```